source/features/show-names.tsx

```diff
@@ -8,94 +8,78 @@ import api from '../github-helpers/api.js';
 import {getUsername, isUsernameAlreadyFullName} from '../github-helpers/index.js';
 import observe from '../helpers/selector-observer.js';
 import {removeTextNodeContaining} from '../helpers/dom-utils.js';
+import {usernameLinksSelector} from '../github-helpers/selectors.js';
 
-function dropExtraCopy(element: HTMLAnchorElement): void {
+async function dropExtraCopy(link: HTMLAnchorElement): Promise<void> {
```

Comment:
Non-awaited sync functions are a quick way to throw errors without stopping execution.

#7472 will again show the feature name related to the error